docker/Dockerfile.centos-8

```diff
@@ -60,6 +60,9 @@ RUN dnf install -y \
         xfsprogs \
     && dnf clean all
 
+# Create vdsm user with groups kvm,qemu,sanlock
+RUN useradd --system -m -u 36 -N -G 36,107,179 vdsm
```

Comment:
This like the only change needed in this pr, added in all dockerfiles.

Reply:
👍